sdk/storage/Azure.Storage.Files/src/ClientHelper.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+using System.Text;
+
+namespace Azure.Storage.Files
+{
+    internal class ClientHelper
```

Comment:
We could rename this.

Reply:
Why does it need to be a partial class?

Reply:
Depending on settings, the code generator may also dump out a `partial static class FileExtensions`

Reply:
fixed.